index.html

```diff
@@ -918,15 +918,21 @@
     </section>
     <section> <h3>Payment scenarios</h3>
       <p>
-        The user buys goods in a store, and payments options include NFC.
+        Payment scenarios with Web NFC generally do not refer to supporting
+        the payment process itself, but recording the fact of payment with a web
+        page in order to have secondary actions. For instance,
+        the user buys goods in a store, and payments options include NFC.
         In general, touching the device to the point of sales terminal receiver
         area will result in a transaction between the secure element from the
         device and the point of sales terminal. With the Web NFC API, if the
         user navigates to a web site before paying, there may be interaction
         with that site regarding the payment, e.g. the user could get points and
         discounts, or get delivered application or service specific data (e.g.
         tickets, keys, etc) to the device.
-        This use case is not supported in this version of the specification.
+      </p>
+      <p class="note">
+        This use case is not supported in this version of the specification (see
+        also <a href="https://github.com/w3c/web-nfc/issues/131">issue 136</a>).
```

Comment:
although issue 131 is not a full coverage of not supporting payment use cases in the spec